packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -722,15 +723,15 @@ class StackViewport extends Viewport {
       ? ViewportStatus.PRE_RENDER
       : ViewportStatus.LOADING;
 
-    if (this.globalDefaultProperties == null) {
-      this.setDefaultProperties({
-        colormap,
-        voiRange,
-        VOILUTFunction,
-        invert,
-        interpolationType,
-      });
-    }
+    this.globalDefaultProperties = {
```

Comment:
Can you explain why you use the global defaults here in preference to the specified property?  This isn't at all clear to me.